### ISSUE TITLE
Ps2 config file issue

### ISSUE THREAD

**Issue by 5yst3mm** [aldostools/webMAN-MOD#828]
Hello, so I have been trying to apply a config file in order to fix a bug in Scooby-Doo First Frights ps2 game which has big graphical issues on the character models without it. I have downlaoded the matching config file from the config database, put it in the ps2iso foldier in hdd0 and renamed it so that it matches the iso game's name exactly. After refeshing the menu and the webman files from my cfw ps3 i mount the game but nothing changes. The thing is that i am not sure if the config is broken or it isnt being applyed. The game is in US region and the config file is meant to fix this specific version. My ps3 might be europe, idk if it matters, im new to this config thing. Let me add that the ps2 netemu option doesnt show in the webman setup, probably cause i have phat ps3. Any suggestions?

**Comment by aldostools** [comment]
Try pressing R2 while mounting the game to select ps2_netemu.

Phat models use by default the hardware emulation, while the CONFIG files are intended for ps2_netemu (software emulation)

**Comment by 5yst3mm** [comment]
At what state do i press R2? I have selected the game from inside the webman menu and i returned to xmb and i see the blue disc so i guess its mounting, and then I tried pressing R2 before and after i launch. I can't find any menu or something. Probably i didn't understand well what you meant.

**Comment by 5yst3mm** [comment]
Let me mention that according to compatibility thread this specific game has issues inspite applying the config file as i said on twitter but id rather get it applied myself to begin with, based on your instructions

**Comment by aldostools** [comment]
Hold R2 then select the PS2 game, and keep pressing R2 until the game is launched.

If your console is a Phat B/C model with CFW Evilnat 4.89 or Rebug 4.84 and Cobra is enabled, the game will be launched with ps2_netemu instead of the default hardware emulator.

**Comment by aldostools** [comment]
IIRC the config file for that game was made my mrjaredbeta or agrippa.
Please report your results in this thread to get their feedback:
https://www.psx-place.com/threads/compatibility-list-ps2-on-ps3.1306/page-285

**Comment by 5yst3mm** [comment]
Yeah it is a phat with 4.89 cobra. I tried what you said but still same results corrupted character models.. So i guess ill proceed reporting those results as i think i haven't done anything wrong myself and its up to the config file

**Comment by pachucojafo** [comment]
I cannot activate PS2netemu through webman, tried everything. its no longer listed in the setup menu either (check box)
I'm running a cechE and would like this option to work, it did in the past.
any help would be appreciated.

**Comment by aldostools** [comment]
Verify if you are spoofing the IDPS. The opton `ps2_netemu` only shows in `/setup.ps3` if the [Product Sub Code](https://www.psdevwiki.com/ps3/Product_Sub_Code) in EID0 IDPS is 01-04. It is stored at 7th and 8th bytes of [IDPS](https://www.psdevwiki.com/ps3/IDPS)
```
Example of IDPS:
00 00 00 01 00 89 00 0B 14 00 EF DD CA 25 52 66
Sub Code-------------^^
0x01 = CECH-A
0x02 = CECH-B
0x03 = CECH-C
0x04 = CECH-E

**Comment by pachucojafo** [comment]
Life saver!
THANK YOU Aldo!!
I would have never figured that out

Thanks again
Rob
